src/containers/DefaultLayout/DefaultLayout.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import * as router from 'react-router-dom';
+import { Container } from 'reactstrap';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Container' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)